### PR TITLE
Added parameter to ignore provider's error during booking cancellatio…

### PR DESCRIPTION
**Pull request by Raibo**

### CODE REVIEW

File: Api/Controllers/AgentControllers/AccommodationsController.cs [hunk 1]
```diff
@@ -270,6 +275,27 @@ public async Task<IActionResult> CancelBooking(int bookingId)
         }
 
 
+        /// <summary>
```
Comment (mingazhev) [inline Api/Controllers/AgentControllers/AccommodationsController.cs:278]: This method should be placed inside the admin controllers folder, maybe create a separate controller for it

File: Api/Controllers/AgentControllers/AccommodationsController.cs [hunk 2]
```diff
@@ -270,6 +275,27 @@ public async Task<IActionResult> CancelBooking(int bookingId)
...
+        ///     Cancel accommodation booking by admin.
+        /// </summary>
+        /// <param name="bookingId">Id of booking to cancel</param>
+        /// <param name="ignoreProviderError">If a provider returns an error after cancellation request, this is ignored as if it was a success</param>
+        /// <returns></returns>
+        [HttpPost("accommodations/bookings/{bookingId}/cancel-by-admin")]
```
Comment (mingazhev) [inline Api/Controllers/AgentControllers/AccommodationsController.cs:284]: After moving just "cancel" verb will be enough

File: Api/Services/Accommodations/Bookings/BookingService.cs [hunk 3]
```diff
@@ -296,8 +296,18 @@ async Task NotifyAgent()
 
             return await ProcessBookingCancellation(booking, serviceAccount.ToUserInfo());
         }
-        
-        
+
+
+        public async Task<Result<VoidObject, ProblemDetails>> Cancel(int bookingId, Administrator administrator, bool ignoreProviderError)
```
Comment (kirillta) [inline Api/Services/Accommodations/Bookings/BookingService.cs:301]: The name `ignoreProviderError` is unclear. May be `requireExistingBooking` or `doesBookingExist`. Also not clear when I must use the condition

File: Api/Models/Management/Enums/AdministratorPermissions.cs [hunk 4]
```diff
@@ -17,5 +17,6 @@ public enum AdministratorPermissions
         CounterpartyManagement = 512,
         PaymentLinkGeneration = 1024,
         AccommodationDuplicatesReportApproval = 2048,
+        BoookingCancellation = 4096,
```
[the remark this answers is not in the document]
Comment (Raibo) [reply Api/Models/Management/Enums/AdministratorPermissions.cs:20]: I thought we don't store any admin permissions yet

File: Api/Services/Accommodations/Bookings/BookingService.cs (hunk 3 above)
Comment (Raibo) [reply Api/Services/Accommodations/Bookings/BookingService.cs:301]: This flag has nothing to do with booking exists or not, it tells whether pipeline should fail on connector error after cancellation attempt or not.
Can't really think of any better name here.

File: Api/Models/Management/Enums/AdministratorPermissions.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (kirillta) [reply Api/Models/Management/Enums/AdministratorPermissions.cs:20]: Ok

File: Api/Services/Accommodations/Bookings/BookingService.cs (hunk 3 above)
Comment (kirillta) [reply Api/Services/Accommodations/Bookings/BookingService.cs:301]: At least it must explain why a booking requires and why I want skip it

**Pull request merged**